TF_Settings_Web/src/Styles/Controls/TFButton.scss

```diff
@@ -0,0 +1,64 @@
+.tf-button {
+    background: transparent linear-gradient(180deg, #373737 0%, #2c2c2c 100%) 0% 0% no-repeat padding-box;
```

Comment:
It could be good to pull the shadow gradient colours into variables so it's we can keep them consistent across the UI